src/org/traccar/protocol/Gps103ProtocolDecoder.java

```diff
@@ -102,6 +102,23 @@ public Gps103ProtocolDecoder(Gps103Protocol protocol) {
             .any()
             .compile();
 
+    private String decodeAlarm(String value) {
+        if (value == "tracker") {
```

Comment:
Do switch instead of ifs. Also, you can't really compare string like that in Java.